{"componentChunkName":"component---src-templates-module-documentation-template-tsx","path":"/modules/documentation/auxiliary/dos/dns/bind_tsig_badtime","result":{"data":{"moduleMetadataJson":{"id":"ffcab562-0b49-5cbc-ba96-c99276cbc9f5","name":"BIND TSIG Badtime Query Denial of Service","fullname":"auxiliary/dos/dns/bind_tsig_badtime","description":"A logic error in code which checks TSIG validity can be used to\n        trigger an assertion failure in tsig.c.","rank":300,"fields":{"detailsSlug":"/modules/details/auxiliary/dos/dns/bind_tsig_badtime","documentationSlug":"/modules/documentation/auxiliary/dos/dns/bind_tsig_badtime","referencesSlug":"/modules/references/auxiliary/dos/dns/bind_tsig_badtime"},"documentation":null}},"pageContext":{"id":"ffcab562-0b49-5cbc-ba96-c99276cbc9f5"}}}